docs/design/2022-03-03-rc-read-tso-optimization.md

```diff
@@ -0,0 +1,36 @@
+# Read-Consistency Read With Timestamp Check
+
+- Author(s): cfzjywxk
+- Last updated: March. 3, 2022
+- Discussion at:
+
+## Motivation
+
+For the read-consistency isolation level read requests in a single transaction, each will need to fetch a new `ts` to read the latest committed data.
+If the workload is a read-heavy one or the read qps is large, fetching ts each time will increase the query lantecy.
```

Comment:
Does "the read qps is large" means "the read QPS is significantly higher than writes"?